cli/src/commands/install.js

```diff
@@ -284,8 +284,22 @@ async function installNpmLibDefs({
     }
     console.log(`• Searching for ${libdefsToSearchFor.size} libdefs...`);
   } else {
+    let ignoreDefs;
+    try {
+      ignoreDefs = fs
+        .readFileSync(path.join(cwd, libdefDir, '.ignore'), 'utf-8')
+        .replace(/"/g, '')
+        .split('\n');
+    } catch (e) {
```

Comment:
Can we scope the catch to only handle the "couldn't find the file" error and otherwise throw? I'm always leery of just silencing all errors. (JS is much worse about being able to do this, which is a bummer...)